### PR TITLE
Update 01_Data_Model_and_ORM.md

### PR DESCRIPTION
**Pull request by jeffreyguo**
Updated NewsArticle to NewsPage accordingly.

### CODE REVIEW

File: docs/en/02_Developer_Guides/00_Model/01_Data_Model_and_ORM.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ information.
 
 All data tables in SilverStripe are defined as subclasses of [api:DataObject]. The [api:DataObject] class represents a 
 single row in a database table, following the ["Active Record"](http://en.wikipedia.org/wiki/Active_record_pattern) 
-design pattern. Database Columns are defined as [Data Types](data_types_and_casting) in the static `$db` variable 
+design pattern. Database Columns are is defined as [Data Types](data_types_and_casting) in the static `$db` variable
```
Comment (dhensby) [inline docs/en/02_Developer_Guides/00_Model/01_Data_Model_and_ORM.md]: "are is" ?

File: docs/en/02_Developer_Guides/00_Model/01_Data_Model_and_ORM.md [hunk 2]
```diff
@@ -401,7 +401,7 @@ Remove both Sam and Sig..
 		'Surname' => 'Minnée',
 	));
 
-And removing Sig and Sam with that are either age 17 or 43.
+And removing Sig and Sam with that are either age 17 or 74.
```
Comment (dhensby) [inline docs/en/02_Developer_Guides/00_Model/01_Data_Model_and_ORM.md]: 74?

File: docs/en/02_Developer_Guides/00_Model/01_Data_Model_and_ORM.md (hunk 1 above)
Comment (jeffreyguo) [inline docs/en/02_Developer_Guides/00_Model/01_Data_Model_and_ORM.md]: Something wrong here, it's a reverse of https://github.com/silverstripe/silverstripe-framework/pull/4000/files

File: docs/en/02_Developer_Guides/00_Model/01_Data_Model_and_ORM.md (hunk 2 above)
Comment (jeffreyguo) [inline docs/en/02_Developer_Guides/00_Model/01_Data_Model_and_ORM.md]: Something wrong here, it's a reverse of https://github.com/silverstripe/silverstripe-framework/pull/4000/files

File: docs/en/02_Developer_Guides/00_Model/01_Data_Model_and_ORM.md (hunk 1 above)
Comment (dhensby) [inline docs/en/02_Developer_Guides/00_Model/01_Data_Model_and_ORM.md]: Rebase, I think?

**Comment by kinglozzer** [comment]
Squashed and merged in e0627c714826f2e1ead2bddd9e50a4aa052008e8, thanks @jeffreyguo :)

**Pull request closed without merging**